### PR TITLE
Restrict running Nielsen third party only in Australia

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?
Nielsen script (imrworldwide.com) is used as an external tool to measure audience size of our audience for Australia. This runs on all pages, and geo's of the site with/without consent.

The business deems this as a risk in terms of data protection. A First step in improving this would be to introduce a condition that only allows Nielsen to run on Australian IPs.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js [hunk 1]
```diff
@@ -31,12 +31,25 @@ jest.mock('lib/config', () => {
     return Object.assign({}, defaultConfig, {
         get: (path: string = '', defaultValue: any) =>
             path
-                .replace(/\[(.+?)\]/g, '.$1')
```
Comment (sndrs) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js:34]: wow how was this working before?

Comment (sndrs) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js:34]: should the closing `]` really not be escaped?

Comment (ioanna0) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js:34]: it has been marked as useless escaped character by my IDE, so I've tried to remove it and seem to work fine. Made this change only in tests, although exact same code exists in implementation code as well

Comment (sndrs) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js:34]: works for me!

**Review by sndrs — APPROVED** [review]

**Pull request merged**